spec/dummy/config/initializers/filter_parameter_logging.rb

```diff
@@ -3,4 +3,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  :password, :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
+]
```

Comment:
The new default for this for 7, other than `:password` which I carried over from our existing filter.